cms/core/blocks/panels.py

```diff
@@ -3,27 +3,39 @@
 from wagtail import blocks
 
 
-class PanelBlock(blocks.StructBlock):
-    """DS Panel block.
-    https://service-manual.ons.gov.uk/design-system/components/panel
-    https://service-manual.ons.gov.uk/design-system/components/announcement-panel
-    https://service-manual.ons.gov.uk/design-system/components/success-panel
-    https://service-manual.ons.gov.uk/design-system/components/warning-panels.
-    """
-
-    variant = blocks.ChoiceBlock(
-        choices=[
-            ("warn", _("Warning")),
-            ("info", _("Information")),
-            ("announcement", "Announcement"),
-            ("error", "Error"),
-            ("success", "Success"),
-        ],
-        default="warn",
-    )
+class WarningPanelBlock(blocks.StructBlock):
     body = blocks.RichTextBlock(features=settings.RICH_TEXT_BASIC)
-    title = blocks.CharBlock(required=False, label=_("Title (optional)"))
+
+    def get_context(self, value: dict, parent_context: dict | None = None) -> dict:
+        context: dict = super().get_context(value, parent_context)
+        context["variant"] = "warn"
+        return context
+
+    class Meta:
+        template = "templates/components/streamfield/warn_announcement_panel.html"
```

Comment:
let's go with separate templates for each of the panels. this will make the clean up easier (if it will be needed).

Reply:
Done. Additionally, I made a `BasePanelBlock` class to to make things a bit more DRY.